facet-json/src/lib.rs

```diff
@@ -2,7 +2,7 @@
 #![warn(missing_docs)]
 #![warn(clippy::std_instead_of_core)]
 #![warn(clippy::std_instead_of_alloc)]
-#![deny(unsafe_code)]
+// #![deny(unsafe_code)]
```

Comment:
This how you tell a quality PR

Reply:
Nah we're just going to move that to facet-reflect :)